src/bilder/images/consul/deploy.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+
+from pyinfra import host
+
+from bilder.components.baseline.setup import install_baseline_packages
+from bilder.components.caddy.models import CaddyConfig, CaddyPlugin
+from bilder.components.caddy.steps import caddy_service, configure_caddy, install_caddy
+from bilder.components.hashicorp.consul.models.consul import (
+    Consul,
+    ConsulACL,
+    ConsulACLToken,
+    ConsulConfig,
+    ConsulTelemetry,
+)
+from bilder.components.hashicorp.consul.steps import proxy_consul_dns
+from bilder.components.hashicorp.steps import (
+    configure_hashicorp_product,
+    install_hashicorp_products,
+    register_services,
+)
+from bilder.facts import has_systemd  # noqa: F401
+
+VERSIONS = {  # noqa: WPS407
+    "caddy_route53": "v1.1.1",
+    "consul": "1.9.5",
```

Comment:
We should update this to something like `os.environ.get("CONSUL_VERSION", "1.10.0")` so that we can override the version in a Concourse pipeline. That will let us trigger a new AMI build when new versions are released and pass that version number without having to change the code here every time.